pom.xml

```diff
@@ -35,7 +35,7 @@
     <maven-assembly.version>3.1.0</maven-assembly.version>
 
     <!-- Dependencies -->
-    <okio.version>1.15.0</okio.version>
+    <okio.version>2.1.0</okio.version>
```

Comment:
might hold this on 1.16
https://github.com/square/okio/pull/520

Reply:
Meh

Reply:
Picasso was going in on Kotlin for peek. Even if backported I'm not sure we'll change there. Moshi also feels like it would be fine going Kotlin. OkHttp and Retrofit should probably wait a year.

Reply:
In my day job I’m to get a bunch of different dependencies to play nicely together: Guava, gRPC, Vitess, and the Google Cloud Datastore. The experience has left me pretty grumpy!

I’m trying to give our users a little flexibility.

That said, I do really want to do a Moshi 2.0 that’s the same in spirit and execution to Okio 2.0. That should be great!